src/main/java/core/basesyntax/RectangleSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class RectangleSupplier {
+    private static final RandomGeometricValue randomSize = new RandomGeometricValue();
+    private static final ColorSupplier color = new ColorSupplier();
+
+    public Figure getRandomRectangle() {

```

Comment:
The method `getRandomRectangle` should be part of the `FigureSupplier` class in a way that it supports creating different types of figures.